benchmark/Usage.md

```diff
@@ -0,0 +1,138 @@
+A short crash course on how to create benchmarks and use them.
```

Comment:
After I'm done with the changes, this file should be updated.
After all, we got:
- New functionality.
- An example in `BasicBenchmark.cpp`.